x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas.ts

```diff
@@ -165,6 +191,7 @@ export const updateRulesSchema = Joi.object({
   tags,
   to,
   type,
+  threats: threats.optional(),
```

Comment:
I think you can just have `threats` without having to put in `optional()` since its default state is optional.